### PR TITLE
style: update text area

### PR DESCRIPTION
**Pull request by RNAdvani**
Updates text-area used at various places

Issue : https://www.loom.com/share/ad367892b1214cf7942c6ada7cca4e55?sid=9419f327-2ad4-4839-afee-fe09ee635ec7

Fix : https://www.loom.com/share/25d9e82572b44107b8c03c6a71730ca1?sid=32ca59ed-c448-4e32-aaef-48684be71207

I have self-reviewed the code (A decent size PR without self-review might be rejected).
I have added a Docs issue [here](https://github.com/calcom/docs/issues/new) if this PR makes changes that would require a [documentation change](https://docs.cal.com). If N/A, write N/A here and check the checkbox.
I confirm automated tests are in place that prove my fix is effective or that my feature works.

### CODE REVIEW

**Comment by anikdhabal** [comment]
@RNAdvani Can you explain the issue in details?

File: packages/ui/components/editor/Editor.tsx [hunk 1]
```diff
@@ -83,8 +83,10 @@ export const Editor = (props: TextEditorProps) => {
             setFirstRender={props.setFirstRender}
           />
           <div
-            className={classNames("editor-inner scroll-bar", !editable && "!bg-subtle")}
-            style={{ height: props.height }}>
+            className={classNames(
+              "editor-inner scroll-bar !h-[10rem] !resize-none",
```
Comment (Amit91848) [inline packages/ui/components/editor/Editor.tsx:87]: We want the text-area to be resizable, also don't increase the height that much

```suggestion
              "editor-inner scroll-bar !h-[10rem]",
```

File: packages/ui/components/editor/stylesEditor.css [hunk 2]
```diff
@@ -42,6 +42,8 @@
   outline: 0;
   padding: 10px 10px;
   outline: none;
+  hyphens: auto !important;
+  word-wrap: break-word !important;
```
Comment (Amit91848) [inline packages/ui/components/editor/stylesEditor.css:46]: What does these do and why is it !imporatant?

File: packages/ui/components/editor/stylesEditor.css [hunk 3]
```diff
@@ -52,7 +54,7 @@
   font-style: italic;
 }
 
-// h6 is only used for emoji links
+/* // h6 is only used for emoji links */
```
Comment (Amit91848) [inline packages/ui/components/editor/stylesEditor.css:57]:
```suggestion
/* h6 is only used for emoji links */
```

**Review by Amit91848 — DISMISSED** [review]
Thanks for the pr @RNAdvani ! Have some doubts lmk what you think

File: packages/ui/components/editor/Editor.tsx [hunk 4]
```diff
@@ -83,8 +83,10 @@ export const Editor = (props: TextEditorProps) => {
             setFirstRender={props.setFirstRender}
           />
           <div
-            className={classNames("editor-inner scroll-bar", !editable && "!bg-subtle")}
-            style={{ height: props.height }}>
```
Comment (anikdhabal) [inline packages/ui/components/editor/Editor.tsx:87]: Hey, this height props pass and set the editor height. Don't remove it

**Review by anikdhabal — CHANGES_REQUESTED** [review]

File: packages/ui/components/editor/Editor.tsx (hunk 1 above)
Comment (RNAdvani) [reply packages/ui/components/editor/Editor.tsx:87]: the reason i think the text area shouldn't be resizable is it looks like normal html simple text area and not all then majority of the websites have text area of fixed size

File: packages/ui/components/editor/stylesEditor.css (hunk 2 above)
Comment (RNAdvani) [reply packages/ui/components/editor/stylesEditor.css:46]: that breaks the word as in small devices i check if the word is a bit big the whole word is shifted to another line which doesn't look good it seems there are many gaps in our paragraph(s).

File: packages/ui/components/editor/Editor.tsx (hunk 1 above)
Comment (anikdhabal) [inline packages/ui/components/editor/Editor.tsx:87]: Make the height 5rem and ensure it is resizable, so remove !resize-none.

File: packages/ui/components/editor/stylesEditor.css (hunk 2 above)
Comment (anikdhabal) [inline packages/ui/components/editor/stylesEditor.css:46]: The changes are correct

**Review by anikdhabal — CHANGES_REQUESTED** [review]
@RNAdvani Left some blocking comments, pls address them

**Review by anikdhabal — APPROVED** [review]
LGTM!

**Review by Amit91848 — APPROVED** [review]

**Pull request merged**